### DISCUSSION TITLE
How to setup inference parameters like temperature?

### DISCUSSION THREAD

**Discussion by rbhatia46** [imartinez/privateGPT#639] [Q&A]
Hi,
How would one go about setting inference parameteres like temperature, max output seq length, etc?

Thanks

**Comment by sime2408** [comment]
```python 
GPT4All(
    ...
    temp=0.4,
)
...
LlamaCpp(
    ....
    temperature=0.4,
)
```

**Reply by zirubak** [reply]
@sime2408 - which file has that value? I can't find it.

**Reply by sime2408** [reply]
https://github.com/imartinez/privateGPT/blob/main/privateGPT.py#L36
